### ISSUE TITLE
More Header and Footer ADA Issues

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#667]
## LIB20230113-02. Multiple navigation landmarks without aria-labels (Best practice)

**Issue**: The main navigation [landmark should have an aria-label attribute](https://www.w3.org/WAI/ARIA/apg/example-index/landmarks/index.html) to denote its function, and distinguish it from other nav landmarks on the page, such as breadcrumbs or subnavigation. Additionally, the main navigation landmark is marked up with a `<nav>` element and a redundant role=”navigation” attribute. For example:

[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit?pli=1#heading=h.w04unad1ahtl)

## LIB20230113-03. Unnecessary ARIA region role in nav dropdowns (Best practice)

**Issue**: The main nav dropdowns work really well. However, the sub nav unordered link lists include ARIA region roles on the `<ul>`s. The [region role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role) is generic, so it removes the semantics from the unordered list for a screen reader. This role is unnecessary here, causes some minor keyboard interactivity and screen reader issues, and can be removed.

[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit?pli=1#heading=h.froo4wo90i0x)

## LIB20230113-04. Text functioning like headings is not marked up with heading tags (Major)

**Issue**: The text above the links in the footer are [functioning as headings](https://dequeuniversity.com/rules/axe/4.4/heading-order) but are not marked up with heading tags. These should all be marked up as `<h2>`s. For example:

[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit?pli=1#heading=h.dupbq4dwyhzx)
